src/components/gcds-radio/gcds-radio.css

```diff
@@ -0,0 +1,112 @@
+:host {
+  .gcds-radio-wrapper {
+    max-width: 90%;
+    font-family: var(--gcds-font-families-body), sans-serif;
+    font-weight: var(--gcds-font-weights-regular);
+    font-size: var(--gcds-font-sizes-paragraph);
+    line-height: var(--gcds-line-heights-paragraph);
+    color: var(--gcds-radio-default-text);
+    margin: 0;
+    padding: 0;
+    border: 0;
+    transition: color ease-in-out 0.15s;
+
+    position: relative;
+    margin-bottom: var(--gcds-spacing-400);
+    margin-top: 1.75rem;
+
+    &:focus-within {
+      color: var(--gcds-radio-focus);
+    }
+
+    &.gcds-disabled {
+      color: var(--gcds-checkbox-disabled-text);
+
+      input[type=radio]+gcds-label {
+        color: var(--gcds-checkbox-disabled-text);
+        &:before {
+          border-color: var(--gcds-checkbox-disabled-border);
+          background-color: var(--gcds-checkbox-disabled-background);
+          cursor: not-allowed;
+        }
+        &:after {
+          cursor: not-allowed;
+        }
+      }
+    }
+
+    &.gcds-error:not(:focus-within) {
+      color: var(--gcds-radio-destructive);
+
+      input[type=radio] + gcds-label {
+        &:before {
+          border-color: var(--gcds-radio-destructive);
+        }
+        &:after {
+          color: var(--gcds-radio-destructive);
+          background-color: var(--gcds-radio-destructive);
+        }
+      }
+    }
+
+    input[type=radio] {
+      height: var(--gcds-spacing-500);
+      left: 0;
+      opacity: 0;
+      position: absolute;
+      top: 0;
+      width: var(--gcds-spacing-500);
+
+      &:checked + gcds-label:after {
+        opacity: 1;
+      }
+
+      &+ gcds-label {
+        width: fit-content;
+        &:before, &:after {
+          position: absolute;
+          cursor: pointer;
+          border-radius: var(--gcds-spacing-500);
+          content: "";
+        }
+        &:before {
+          border: 0.125rem solid var(--gcds-color-grayscale-1000);
```

Comment:
Sorry, I just have explained this better, but do you mind creating radio specific colour tokens for these?  For example: --gcds-radio-label-checkmark-border or something along these lines.